examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/src/authorizer/lambda_function.py

```diff
@@ -0,0 +1,218 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# IMPORTANT: 
+#     This is mock implementation of a Lambda Authorizer
+#     It allows ALL actions on your API to be performed by ANYONE
```

Comment:
If this is the case, I don't think we should check in unused code.

Reply:
It is not unused code. It generates policy that allows all actions on the resource. Updated comment to make it clearer.